pandas/core/nanops.py

```diff
@@ -326,14 +329,19 @@ def nanmean(values, axis=None, skipna=True):
 @bottleneck_switch()
 def nanmedian(values, axis=None, skipna=True):
 
+    if not skipna and isnull(np.sum(values)):
+        return np.median(values, axis=axis)
```

Comment:
@rohanp Can you deal with this comment? (remove or give some explanation on why you think it is needed)

BTW, `isnull` was recently be renamed to `isna` in this file, that is the reason all tests are failing now.